--- zio-http/shared/src/main/scala/zio/http/Route.scala
+++ zio-http/shared/src/main/scala/zio/http/Route.scala
@@ -439,7 +439,7 @@
   ): Route[Env0, Err] =
     self.transform[Env0](handler => handler @@ aspect)

-  final def @@[Env0](implicit dummy: DummyImplicit): ApplyContextAspect[Env, Err, Env0] =
+  final def @@[Env0](implicit dummy: DummyImplicit, dummy2: DummyImplicit): ApplyContextAspect[Env, Err, Env0] =
     new ApplyContextAspect[Env, Err, Env0](self.toRoutes)

   def toHandler(implicit ev: Err <:< Response, trace: Trace): Handler[Env, Response, Request, Response]
